openmetadata-ui/src/main/resources/ui/src/components/Entity/Task/TaskTab/TaskTab.component.tsx

```diff
@@ -568,7 +568,13 @@ export const TaskTab = ({
                 onClick={() =>
                   handleMenuItemClick({ key: TaskActionMode.EDIT } as MenuInfo)
                 }>
-                {t('label.add-suggestion')}
+                {taskThread.task?.newValue
+                  ? t('label.add-suggestion')
+                  : t('label.add-entity', {
+                      entity: isTaskTags
+                        ? t('label.tag-plural')
+                        : t('label.description'),
+                    })}
```

Comment:
We will only need this for description and not for tags since for tags tasks, newValue always be there as it's required field while creating

Reply:
@chirag-madlani  we can request tags without the tags from the UI 

<img width="628" alt="image" src="https://github.com/open-metadata/OpenMetadata/assets/66266464/d27b9dc3-a2cb-4d1a-937b-8137c0db3aa6">
<img width="792" alt="image" src="https://github.com/open-metadata/OpenMetadata/assets/66266464/1c65ce9d-4f90-48ee-8e1b-221a5b904130">